library/manageiq_provider.py

```diff
@@ -432,16 +437,20 @@ def add_or_update_provider(self, provider_name, provider_type, endpoints, zone,
                                                 endpoints, zone_id, provider_region)
             roles_with_changes = [e['endpoint']['role'] for e in endpoints]
 
-        authtypes_to_verify = []
-        for e in endpoints:
-            if e['endpoint']['role'] in roles_with_changes:
-                authtypes_to_verify.append(e['authentication']['authtype'])
-        result, details = self.verify_authenticaion_validation(provider_id, old_validation_details, authtypes_to_verify)
+        if not skip_refresh_and_validation:
+            authtypes_to_verify = []
+            for e in endpoints:
+                if e['endpoint']['role'] in roles_with_changes:
+                    authtypes_to_verify.append(e['authentication']['authtype'])
+            result, details = self.verify_authenticaion_validation(provider_id, old_validation_details, authtypes_to_verify)
+        else:
+            result = "Valid"
```

Comment:
I don't think `"Valid"` is the right term to describe the result of the authentication in case it was skipped.
Consider adding another option of `"Skipped"` and:

``` python 
elif result == "Skipped":
```